platform/pulpcore/app/status.py

```diff
@@ -0,0 +1,65 @@
+"""
+Module to return status information about a running Pulp instance
```

Comment:
+1 to moving them into the view. This code is dedicated to that view and not used otherwise.